srcpkgs/llvm15/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'llvm15'
 pkgname=llvm15
 version=15.0.7
-revision=2
+revision=1
```

Comment:
I'm relatively new in void linux so don't beat me with keyboards pls :) Why do we want to drop `revision` here? I think it should be only incremental otherwise it could mess up with indices. If someone has llvm-15.0.7-1 and update the indices it would see the same name-version-revision package but with a different hash as well as the folks with 15.0.7-2 will get into trouble.

Reply:
Good catch, this is a mistake